blamo/lib/Export/email.dart

```diff
@@ -0,0 +1,45 @@
+import 'dart:async';
+import 'package:blamo/ObjectHandler.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:flutter_email_sender/flutter_email_sender.dart';
+import 'package:permission_handler/permission_handler.dart';
+
+
+//Function to open ACTION SHARE activity with email sender
+// Needs docName and extension type csv or pdf with future option of having it zipped
+// Will throw exception if pdf/csv filepath is not found
+// TODO need to create backend to vacuum up all doc files and create pdf/csv based only on doc name
+Future<String> sendEmail(String documentName,String docType) async {
+  ObjectHandler objectHandler = new ObjectHandler();
+  //String outputPath = "storage/emulated/0/Android/data/edu.oregonstate.blamo/files/output_test.pdf";
+  String outputPath;
+  debugPrint("EMAIL.dart - Send email");
+
+  outputPath = await objectHandler.getPathToFile(documentName, docType);
+  debugPrint("output path: " + outputPath);
```

Comment:
Getting exception ArgumentError (Invalid argument(s)) here on line 19 in following cases
- csv email without making csv
- pdf email without making pdf
- zip both without making either
- csv email after making both
- zip both after making both

Was able to email pdf after making that though.